docs/vsphere_ci/scripts/install-docker.ps1

```diff
@@ -9,3 +9,5 @@ Set-PSRepository PSGallery -InstallationPolicy Trusted
 Install-Module -Name DockerMsftProvider -Repository PSGallery -Force
 # install docker package
 Install-Package -Name docker -ProviderName DockerMsftProvider -Force
```

Comment:
Shouldnt we not be doing this for 2022?

Reply:
+1, we should not be doing this.
if we are backporting this into 5.x, then we need to add it backport it in 5.x and other branches that we are trying to backport.

Reply:
Since this is going into 5.1.0, we will have to have docker installed. When docker support is dropped in 6.0.0 then we will have to build a new golden image dropping the docker installation (with [containers feature explicit enabled](https://github.com/openshift/windows-machine-config-operator/pull/1035/commits/1ed065116cf1e0a0eabb7735011a68f3e6f57631?short_path=940babd#r862193771)).